### DISCUSSION TITLE
Help - Implement Mosaico into SuiteCRM

### DISCUSSION THREAD

**Discussion by James-van-der-Hoven** [voidlabs/mosaico#660] [Q&A]
Similar to what CiviCRM has done to implement Mosaico into Civi, we would like to be able to implement Mosaico into SuiteCRM 8 ([see forum post here](https://community.suitecrm.com/t/mosaico-for-email-newsletters/85151))

Can the Mosaico team help with this or collaborate with ourselves and the SuiteCRM team to get this up and running please?

The end result must be that each of our clients get their own dashboard and from there, be able to send their own emails campaigns.

Many thanks

**Comment by bago** [comment]
CiviCRM people did the integration on their own.
You can have a look to our wiki and use the "Discussions" tab if you have specific questions.
